docs/source/install.rst

```diff
@@ -1,15 +1,51 @@
 Install Dask
 ============
 
-You can install dask using ``conda``::
+Dask is available to Anaconda users via either the conda or pip commands, both of which are included in Anaconda. If you do not already have Anaconda, `download Anaconda now <https://www.continuum.io/downloads>`_.
```

Comment:
This sounds like an advertisement for Anaconda.  This makes me very uncomfortable.  

We should definitely link to Anaconda at some point, but not with this level of prominence.

Also, in case it's not clear.  Dask is available to all Python users, even those that don't use Anaconda.